src/main/java/racingcar/model/InputValidator.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.model;
+
+public class InputValidator {
+
+    public static void validateTrialInput(String input) {
+        try {
+            isPositive(input);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(ErrorMessage.TRIAL_INPUT_IS_NOT_NUMBER);
```

Comment:
NumberFormat 예외를 catch 해서 IllegalArgument 예외를 던진 것이 맞다면, 예외를 바꿔서 던질 때에는 이전 예외에 대한 스택 트레이스가 함께 던져져야 예외 발생 원인을 추적하기 쉽다고 합니다..

```suggestion
        throw new IllegalArgumentException(ErrorMessage.TRIAL_INPUT_IS_NOT_NUMBER, e); // 원본 예외 포함
```

GPT 이용해서 답변 드리는데 어떻게 생각하시나요??

Reply:
해당 내용은 처음봐서 신기하네요!! 공부하겠습니다! 감사합니다!!

Reply:
저도 이 부분 배워갑니다